### PR TITLE
UGENE-7827 Primer library Import button is disabled

### PR DESCRIPTION
**Pull request by DmitriiSukhomlinov**

### CODE REVIEW

File: src/plugins/GUITestBase/src/runnables/ugene/plugins/pcr/ImportPrimersDialogFiller.cpp [hunk 1]
```diff
@@ -47,15 +50,17 @@ ImportPrimersDialogFiller::ImportPrimersDialogFiller(HI::GUITestOpStatus& os, Cu
...
 #define GT_METHOD_NAME "commonScenario"
 void ImportPrimersDialogFiller::commonScenario() {
-    if (fileList.isEmpty()) {
+    if (fileList.isEmpty() && objectNameList.isEmpty()) {
         GTUtilsDialog::clickButtonBox(os, QDialogButtonBox::Cancel);
         return;
     }
 
-    foreach (const QString& file, fileList) {
+    for (const QString& file : fileList) {
```
Comment (yalgaer) [inline src/plugins/GUITestBase/src/runnables/ugene/plugins/pcr/ImportPrimersDialogFiller.cpp]: use qAsConst

File: src/plugins/pcr/src/import/ImportPrimersDialog.cpp [hunk 2]
```diff
@@ -115,14 +115,17 @@ void ImportPrimersDialog::sl_selectionChanged() {
...
 
 void ImportPrimersDialog::sl_contentChanged() {
-    buttonBox->button(QDialogButtonBox::Ok)->setEnabled(lwFiles->count() > 0);
+    buttonBox->button(QDialogButtonBox::Ok)->setEnabled(lwFiles->count() > 0 || lwObjects->count() > 0);
 }
 
 void ImportPrimersDialog::accept() {
     QList<Task*> tasks;
-    foreach (const QString& filePath, item2file) {
+    for (const QString& filePath : item2file) {
```
Comment (yalgaer) [inline src/plugins/pcr/src/import/ImportPrimersDialog.cpp]: Same here: use qAsConst with for & qt collections

**Review by yalgaer — COMMENTED** [review]
Looks good, please fix for loop issues and I will approve after green tests

File: src/plugins/GUITestBase/src/runnables/ugene/plugins/pcr/ImportPrimersDialogFiller.cpp (hunk 1 above)
Comment (DmitriiSukhomlinov) [reply src/plugins/GUITestBase/src/runnables/ugene/plugins/pcr/ImportPrimersDialogFiller.cpp]: OK

File: src/plugins/pcr/src/import/ImportPrimersDialog.cpp (hunk 2 above)
Comment (DmitriiSukhomlinov) [reply src/plugins/pcr/src/import/ImportPrimersDialog.cpp]: OK

**Review by yalgaer — APPROVED** [review]

**Pull request merged**